test/mix/tasks/phx.new_test.exs

```diff
@@ -92,8 +92,8 @@ defmodule Mix.Tasks.Phx.NewTest do
 
   test "assets are in sync with installer" do
     for file <- ~w(favicon.ico phoenix.js phoenix.png) do
-      assert File.read!("priv/static/#{file}") ==
-             File.read!("installer/templates/phx_static/#{file}")
+      assert File.read!("priv/static/#{file}") |> String.replace("\n", "") ==
+             File.read!("installer/templates/phx_static/#{file}") |> String.replace("\n", "")
```

Comment:
@chrismccord This is failing b/c a new line at the end of one of them.  This is reproducible on master.  So I regenerated the build and copied it to the `installer` (not sure if you have an automated way to do this or we should build that in).  I can revert this step if you would like.

<img width="586" alt="Screen Shot 2020-03-22 at 1 26 54 PM" src="https://user-images.githubusercontent.com/7374640/77259723-d118ed80-6c40-11ea-8582-361cededc764.png">

Reply:
removed from PR